image.yaml

```diff
@@ -30,11 +30,11 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-18
+                  ref: master
```

Comment:
Due to https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/99 coming in late, after I realized we needed to fix the buildability of the CD 12 tag.